src/msw/window.cpp

```diff
@@ -2676,7 +2676,104 @@ bool wxWindowMSW::MSWProcessMessage(WXMSG* pMsg)
         }
     }
 #else // __WXUNIVERSAL__
-    wxUnusedVar(pMsg);
+    // But it also handles clicks on default button
```

Comment:
It looks like the code here is not very different from the code in the `#ifndef __WXUNIVERSAL__` branch and it's not ideal to have 2 similar versions of the same code here. Could this be avoided somehow?

If it really can't, it would be better to override `MSWProcessMessage()` in wxUniv to do its own thing there.